.github/scripts/get_latest_runtime_image_tag.py

```diff
@@ -0,0 +1,38 @@
+"""This script fetches the latest runtime image tag from the provided Quay.io repository URL."""
+from __future__ import annotations
+
+import sys
+
+import requests
+from semantic_version import Version
+
+
+def get_latest_tag(repository: str) -> str:
+    """Get the latest semantic version tag from a Quay.io repository."""
+    url = f"https://quay.io/api/v1/repository/{repository}/tag/"
+    response = requests.get(url)
+    response.raise_for_status()
+    data = response.json()
+    tags = data["tags"]
+    valid_tags = []
+    for tag in tags:
+        try:
+            if not tag["name"].endswith("-base"):
+                continue
+            version = Version(tag["name"])
+            valid_tags.append(version)
+        except ValueError:
+            continue
+    if valid_tags:
+        _latest_tag = max(valid_tags)
```

Comment:
I think max() will fail to compare versions like `1.2.2` < `2.1.1`. How about using `distutils.version.StrictVersion` to get the max version instead?

Reply:
This won't fail. Do you have different example in mind that could fail?
<img width="421" alt="Screenshot 2023-12-26 at 12 57 01 PM" src="https://github.com/astronomer/astro-sdk/assets/10206082/f6504979-9019-429a-a9a4-d6e5d45b38e3">

Reply:
Here, we already convert it to Version object https://github.com/astronomer/astro-sdk/pull/2080/files#diff-4381c7dd83cf01b553505437a870a4fb0da096c814049616aaf8b8d2750d4c06R22

Reply:
right this file is from astronomer-provider repo